backend/src/schema/user.ts

```diff
@@ -120,8 +120,7 @@ const resolvers = {
         email: gitHubUser.email,
         reposurl: gitHubUser.repos_url,
       }
-
-      const token = createToken(context.currentUser, access_token.access_token)
+      const token = createToken(context.currentUser, access_token.toString())
```

Comment:
Very hard to test this before our local loginform is integrated but if this is the correct way, then I think we need to fix the type of `access_token` in the first place. Is it really like `GitHubAccessTokenResponse` says or like `GitHubAccessToken`

Reply:
`access_token.access_token` was definitely undefined, so yes I think the type has to be changed

Reply:
This is now fixed.